packages/cra-template-typescript/template.json

```diff
@@ -40,36 +30,29 @@
       "loglevel": "1.x",
       "react-fela": "11.x",
       "react-final-form": "6.x",
-      "react-final-form-redux-submit": "^2.0.0",
-      "react-intl": "3.x",
+      "react-final-form-redux-submit": "2.0.0",
+      "react-intl": "5.24.1",
       "react-redux": "7.x",
       "react-router-dom": "5.x",
       "redux": "4.x",
       "redux-promise-listener": "1.x",
       "redux-saga": "1.x",
-      "reselect": "4.x"
+      "reselect": "4.x",
+      "@testing-library/jest-dom": "^5.14.1",
```

Comment:
There are some devDependencies mixed in to dependencies

Reply:
good point